woocommerce/woocommerce
# Beta Tester: Update WCA Test Helper option modal to show full option name

### All Submissions:






### Changes proposed in this Pull Request:



This PR updates the WCA Test Helper option modal that is used when editing an option, to:

* Show the full option name
* Provide labels for the fields on the modal
* Provide a Cancel button

These changes:

* Allow the dev/tester to have more confidence that they are editing the correct option, as they can see the full option name
* Make the dialog look more polished (it's a test tool, but it still is a product that has customers!)

**Note: When testing, I ran into issues where some options couldn't be saved... this is unrelated to the changes in this PR and will need to be addressed separately.**

#### Before

<img width="1275" alt="Screenshot 2023-03-29 at 07 21 53" src="https://user-images.githubusercontent.com/2098816/228579609-9207d433-a0fc-4a63-be4d-49c64da34e2e.png">

#### After

<img width="1275" alt="Screenshot 2023-03-29 at 09 38 44" src="https://user-images.githubusercontent.com/2098816/228579653-6051d758-f34b-4a18-a29e-5fa5cdc8b211.png">



### How to test the changes in this Pull Request:




1. Go to the Options screen in WCA Test Helper
2. Click "Edit" button for an option
3. Verify that the modal shows the full option name
4. Verify that the Cancel button closes the modal



### Other information:




### FOR PR REVIEWER ONLY:
